src/server/processLauncher.ts

```diff
@@ -135,7 +135,7 @@ export async function launchProcess(options: LaunchProcessOptions): Promise<Laun
     }
     gracefullyClosing = true;
     debugBrowser(`<gracefully close start>`);
-    options.attemptToGracefullyClose().catch(() => killProcess());
+    await options.attemptToGracefullyClose().catch(() => killProcess());
```

Comment:
Is it necessary?

Reply:
if killProcess throws, the error will appear in the right place, among other subtle changes. I always prefer to await promises.